ecosystem/indexer-grpc/indexer-grpc-table-info/src/runtime.rs

```diff
@@ -13,6 +13,25 @@ use tokio::runtime::Runtime;
 
 const INDEX_ASYNC_V2_DB_NAME: &str = "index_indexer_async_v2_db";
 
+pub fn bootstrap_db_tailer(
+    config: &NodeConfig,
+    db_rw: DbReaderWriter,
+) -> Option<(Runtime, Arc<DBIndexer>)> {
+    if !(config.index_db_tailer.enable_event() || config.index_db_tailer.enable_transaction()) {
+        return None;
+    }
+    let runtime = aptos_runtimes::spawn_named_runtime("index-db".to_string(), None);
+    // Set up db config and open up the db initially to read metadata
+    let mut tailer_service = TailerService::new(db_rw.reader, config);
+    let db_tailer = tailer_service.get_db_tailer();
+    // Spawn the runtime for db tailer
+    runtime.spawn(async move {
+        tailer_service.run().await;
+    });
+
```

Comment:
nit this is the same with `runtime.spawn(tailer_service.run())`?

Reply:
by default, closure borrow and lifetime not long enough.

Reply:
oh... nice